server/routes/overview/preInductionOverviewView.ts

```diff
@@ -0,0 +1,28 @@
+import type { FunctionalSkills, InPrisonEducationRecords, PrisonerSummary } from 'viewModels'
+
+export default class PreInductionOverviewView {
```

Comment:
New view class, which is very similar to `PostInductionOverviewView`, except it does not have an `ActionPlan` and has its `isPostInduction` property set to `false`